samples/fan_out_fan_in_tensorflow/README.md

```diff
@@ -0,0 +1,51 @@
+# Fan Out Fan In Image Classification
+
+This sample demonstrates how to orchestrate parallel executions of image classification using tensorflow through the Fan In Fan Out Durable orchestration pattern
+
+![architecture_diagram](./FanOutFanIn.jpg) 
```

Comment:
This is really cool! But could we get a higher resolution picture? Seems a little pixelated